src/main/java/org/sigmah/server/mail/MailSender.java

```diff
@@ -22,39 +22,49 @@
  * #L%
  */
 
-
-import java.io.InputStream;
-import org.apache.commons.mail.EmailException;
-
-/**
- * Defines a mail sender.
- * 
- * @author Maxime Lombard (mlombard@ideia.fr)
- * @author Raphaël Calabro (rcalabro@ideia.fr)
- */
-public interface MailSender {
-
-	/**
-	 * Sends the given email.
-	 * 
-	 * @param email
-	 *          The email.
-	 * @throws EmailException
-	 *           If an error occurs during the sending.
-	 */
-	void send(final Email email) throws EmailException;
-
-	/**
-	 * Sends the given email with a file as attached content.
-	 * 
-	 * @param email
-	 *			Email to send.
-	 * @param fileName
-	 *			Name of the attachment.
-	 * @param fileStream
-	 *			Content of the attachment.
-	 * @throws EmailException 
-	 */
-	void sendFile(final Email email, String fileName, InputStream fileStream) throws EmailException;
-	
-}
+
+import java.io.InputStream;
+import org.apache.commons.mail.EmailException;
+
+/**
+ * Defines a mail sender.
+ * 
+ * @author Maxime Lombard (mlombard@ideia.fr)
+ * @author Raphaël Calabro (rcalabro@ideia.fr)
+ */
+public interface MailSender {
+
+	/**
+	 * Sends the given email.
+	 * 
+	 * @param email
+	 *          The email.
+	 * @throws EmailException
+	 *           If an error occurs during the sending.
+	 */
+	void send(final Email email) throws EmailException;
+
+	/**
+	 * Sends the given email with a file as attached content.
+	 * 
+	 * @param email
+	 *			Email to send.
+	 * @param fileName
+	 *			Name of the attachment.
+	 * @param fileStream
+	 *			Content of the attachment.
+	 * @throws EmailException 
+	 */
+	void sendFile(final Email email, String fileName, InputStream fileStream) throws EmailException;
```

Comment:
@Raphcal there is an obvious value parameter pattern (one object containing all arguments for the call): do you know why the sendFile method above takes more parameter and why thoses were not added as field of the `Email` object? I guess @mohamedkha could have just make it take a collection instead of a single value.